app/definitions/ISubscription.ts

```diff
@@ -46,6 +46,7 @@ export interface ISubscription {
 	ls: Date;
 	name: string;
 	fname?: string;
+	sanitizedFname: string;
```

Comment:
You created a new column on db as optional, but here you're declaring as required.